app/views/shared/_location.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if object.location %>
+  <p class="location">
+    <i class="fas fa-map-marker-alt"></i> 
+    <%= object.location["city"] %>, <%= object.location["country_name"] %>
+  </p>
+<% end %>
```

Comment:
awww, no map?

Reply:
Added a new class to render static map images from the Google Map API